hvpy/core.py

```diff
@@ -0,0 +1,27 @@
+import requests
+
+from hvpy.io import HvpyParameters, OutputType
+
+
+def parse_response(response: requests.Response, output_parameters: OutputType):
+    """
+    This function parses the response from the API call based on the output
+    type.
+    """
```

Comment:
Can these be in this format https://numpydoc.readthedocs.io/en/latest/format.html ?